templates/gitlab.rb.j2

```diff
@@ -11,6 +11,9 @@ gitlab_rails['gitlab_email_display_name'] = "{{ gitlab_email_display_name }}"
 gitlab_rails['gitlab_email_reply_to'] = "{{ gitlab_email_reply_to }}"
 {% endif %}
 
+# Default Theme
+gitlab_rails['gitlab_default_theme'] = "{{ gitlab_default_theme | default('2') }}"
```

Comment:
Instead of adding the default filter, let's just add it as a default in `defaults/main.yml`, and change this to just `{{ gitlab_default_theme }}`.